components/dashboard/script.js

```diff
@@ -12,7 +12,9 @@ let vars = {
 };
 
 let data = "";
-for(let prop in vars) data += prop + "=" + vars[prop] + "\n";
+for(let prop in vars) {
```

Comment:
<details>
<summary>:warning: <strong>This code uses `let` or `var` to declare one or more variables that are never reassigned.</strong> (click for details)</summary>

A possible improvement would be to declare such a variable with `const` instead.   
  
This change would ensure that it would not be reassigned unintentionally, which could lead to difficulty to comprehend bugs.

**[More details](https://confluence.devfactory.com/x/UA_kEw)**

## The improvement suggestion in this case
```js  
for (const prop in vars) {
```  
> NOTE: Consider applying the structure of this solution in other parts of the code, if there is another occurrence.
</details>

[//]: # (<a id="8c04c96f-a100-47a4-9efa-25a078d652fc" data-finding-id="329518928" href="none" codereview-uuid-reserved-tag data-business-id="a365ef85-720e-4b91-8367-2fe7ee0de9a5" data-analyzer="js" />)